### DISCUSSION TITLE
Change Overlay ScrollBars' theme on site's theme change

### DISCUSSION THREAD

**Discussion by loctran016** [KingSora/OverlayScrollbars#367] [Q&A] [answered]
Hi, there. I'm using TailwindCSS and DaisyUI (with a lot of themes). I'm trying to switch the Overlay ScrollBars' theme between `os-theme-light` and `os-theme-dark` but haven't succeed yet.
Here is the link to my work on [Codepen](https://codepen.io/loctran016/pen/qBrwOmg).
Here is my [index.html](https://github.com/cokaps016/GenZOC/blob/main/src/index.html#L31-L65) and [scrollBar](https://github.com/cokaps016/GenZOC/blob/main/src/js/scrollBar.js)
Thanks in advance

**Comment by KingSora** [comment, marked as answer]
Good day @cokaps016 

You have to change the OverlayScrollbars instance options after each theme select. I've managed to create a example: https://codepen.io/kingsora-the-decoder/pen/eYvodPY

**Comment by KingSora** [comment]
@cokaps016 does this answer your question?

**Reply by loctran016** [reply]
Oh,sort for not telling you. It works well. Thanks for your help really much.
